vcd/resource_vcd_org.go

```diff
@@ -34,44 +39,53 @@ func resourceOrg() *schema.Resource {
 				ForceNew: false,
 			},
 
-			"is_enabled": &schema.Schema{
-				Type:     schema.TypeBool,
-				Required: true,
+			"description": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
 				ForceNew: false,
 			},
+			"is_enabled": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				ForceNew:    false,
+				Default:     false,
```

Comment:
Shouln't organisation be enabled by default?

Reply:
Originally, this was a mandatory property.  I can either revert it to `Required` or change the default as `true`. Which one would be preferred?

Reply:
Let's make Org enabled by default, which will lead to a usable configuration with less parameters.